package.json

```diff
@@ -15,6 +15,8 @@
   "dependencies": {
     "js-beautify": "^1.6.14",
     "optimist": "^0.6.1",
+    "uglify-js": "^3.1.3",
```

Comment:
Doesn't look like you're using this anywhere, why is it in dependencies?

Reply:
The `min` task uses uglifyjs. I ran `npm run dist` for something and it complained about not having uglify-js`. I guess I can add it as a dev-dependency ( like so: https://github.com/dogescript/dogescript/blob/master/package.json)